component/audioinput/v1/audioinput.pb.gw.go

```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/utilities"
-	"go.viam.com/api/common/v1"
+	v1_0 "go.viam.com/api/common/v1"
```

Comment:
Should these changes be showing up in this PR?

Reply:
The answer appears to be yes as that was done by the CI system: https://github.com/viamrobotics/api/pull/226/commits

I also confirmed with james here: https://viaminc.slack.com/archives/C01Q5S8V6TB/p1680025534478849?thread_ts=1680018444.526449&cid=C01Q5S8V6TB

Reply:
Ah I see, it gets added by CI to the PR itself after (one?) approval. Thank you!